bin/latexmlmath

```diff
@@ -240,12 +240,14 @@ if ($xmath) {
   outputXML($result->findnode('//ltx:XMath'), $xmath); }
 
 my $status = $latexml->getStatusMessage;
+my $code = $latexml->getStatusCode;
 # Should be combined with $post's status!
 # But better approach will be to manage all through LaTeXML.pm!!
 my $runtime = RunTime($starttime);
 NoteLog("Conversion complete: " . $status);
 NoteSTDERR("Conversion complete: " . $status . " (reqd. $runtime)");
 UseLog(undef);
+exit $code;
```

Comment:
ah, but that's not the same convention. If I remember correctly the exit code has to be zero in all cases except when the run hits a Fatal. `getStatusCode` returns our own internal levels (0,1,2,3) for (ok, warning, error, fatal). So if you want to use that, you'd likely exit with:

```perl
exit $code == 3 ? 1 : 0;
```

Reply:
Done, fixed both `latexmlmath` and `latexmlpost`